home layout refinement

gozart1 opened the pull request:
This PR explores alternative possibilities for handling homepage layout in
https://github.com/decred/dcrdata/pull/961

Note this PR bumps up the boostrap grid to 24 cols, so would require updating the rest of the views if adopted. 

Mainly I'd like to find ways to make the layout simple, more uniform and space efficient

<img width="1280" alt="screen shot 2019-01-31 at 2 25 01 am" src="https://user-images.githubusercontent.com/25571523/52048404-286d7300-2500-11e9-9901-5df23dd89d79.png">
<img width="1274" alt="screen shot 2019-01-31 at 2 23 24 am" src="https://user-images.githubusercontent.com/25571523/52048408-2b686380-2500-11e9-9b2d-1733ccd7d64e.png">

jholdstock commented:
I think this looks really, really great. 

My main sources of info tend to be dcrdata for block/tx data, and dcrstats.com for PoS/PoW info.

This kind of change would really make dcrdata a true contender for replacing dcrstats.com - something I have wanted to see for a long time :)

chappjc commented:
@gozart1 You can rebase just 412f311 on master now that #961 is in.

jholdstock commented:
A tooltip (or similar) might be nice to explain how the exchange rate is being calculated.

I'm probably wrong, but I believe this is the first time dcrdata has shown information from a source other than the decred blockchain. It would be nice to know what the data source is, especially if it is something like fiat pricing. Fiat price can be quite a sensitive topic for some people, so it's good to be open about where the number is coming from

gozart1 commented:
Rebased.
- updated mempool transaction label colors to use brand colors ( kept the pink though b/c there's not enough contrast within the 6 brand colors for 4 distinct labels )
- refined layout and typography slightly to improve mobile viewing

Wide screen
![screen shot 2019-02-02 at 1 57 48 am](https://user-images.githubusercontent.com/25571523/52163029-cf811480-2690-11e9-9c86-cae033e14f8d.png)
Narrow screen
![screen shot 2019-02-02 at 1 58 28 am](https://user-images.githubusercontent.com/25571523/52163030-d3149b80-2690-11e9-9f52-e89f30e4e2e5.png)
Mobile
![screen shot 2019-02-02 at 2 29 11 am](https://user-images.githubusercontent.com/25571523/52163108-65696f00-2692-11e9-81fc-de5a45ea2970.png)

xaur commented:
> This kind of change would really make dcrdata a true contender for replacing dcrstats.com - something I have wanted to see for a long time :)

Me too. What's lacking for me to replace dcrstats is clear recent tix price chart and clear recent price chart. I open dcrstats to see if it's a good time to buy tix, and tix price chart on the main page saves time.
